soroban-sdk/src/tests/crypto_bls12_381.rs

```diff
@@ -0,0 +1,140 @@
+use crate::{
+    bytes, bytesn,
+    crypto::{Bls12_381, Fp, Fp2, G1Affine, G2Affine},
+    vec, Bytes, Env, Vec, U256,
+};
+
+#[test]
+fn test_bls_g1() {
```

Comment:
It seems like tests are missing for `checked_` ops.

Reply:
good catch, I added the test case, for now only for the happy path just to check the interface, I can later add some cases for invalid point (they do exist in the `env`).